test/passenger_test.rb

```diff
@@ -69,6 +77,144 @@
   end
 
   describe "net_expenditures" do
-    # You add tests for the net_expenditures method
+    before do
+      
+      @passenger = RideShare::Passenger.new(
+        id: 9,
+        name: "Merl Glover III",
+        phone_number: "1-602-620-2330 x3723",
+        trips: []
+        )
+
+      driver = RideShare::Driver.new(
+        id: 54,
+        name: "Rogers Bartell IV",
+        vin: "1C9EVBRM0YBC564DZ",
+        status: :AVAILABLE      
+        )
+  
+      @trip1 = RideShare::Trip.new(
+        id: 8,
+        driver: driver,
+        passenger: @passenger,
+        start_time: Time.new(2016, 8, 8),
+        end_time: Time.new(2016, 8, 9),
+        cost: 10,
+        rating: 5
+        )
+
+      @trip2 = RideShare::Trip.new(
+        id: 10,
+        driver: driver,
+        passenger: @passenger,
+        start_time: Time.new(2016, 8, 11),
+        end_time: Time.new(2016, 8, 12),
+        cost: 40,
+        rating: 5
+        )
+
+      @trip3 = RideShare::Trip.new(
+        id: 11,
+        driver: driver,
+        passenger: @passenger,
+        start_time: Time.new(2016, 8, 11) 
+        )
+      end
```

Comment:
Incorrect indentation:

```suggestion
    before do
      
      @passenger = RideShare::Passenger.new(
        id: 9,
        name: "Merl Glover III",
        phone_number: "1-602-620-2330 x3723",
        trips: []
      )

      driver = RideShare::Driver.new(
        id: 54,
        name: "Rogers Bartell IV",
        vin: "1C9EVBRM0YBC564DZ",
        status: :AVAILABLE      
      )
  
      @trip1 = RideShare::Trip.new(
        id: 8,
        driver: driver,
        passenger: @passenger,
        start_time: Time.new(2016, 8, 8),
        end_time: Time.new(2016, 8, 9),
        cost: 10,
        rating: 5
      )

      @trip2 = RideShare::Trip.new(
        id: 10,
        driver: driver,
        passenger: @passenger,
        start_time: Time.new(2016, 8, 11),
        end_time: Time.new(2016, 8, 12),
        cost: 40,
        rating: 5
      )

      @trip3 = RideShare::Trip.new(
        id: 11,
        driver: driver,
        passenger: @passenger,
        start_time: Time.new(2016, 8, 11) 
      )
    end
```